pkg/lint/lintersdb/validator_test.go

```diff
@@ -215,3 +216,25 @@ func TestValidator_validatePresets_error(t *testing.T) {
 		})
 	}
 }
+
+func TestValidator_alternativeNamesDeprecation(t *testing.T) {
+	log := logutils.NewMockLog().
+		OnWarnf("The linter name %q is deprecated. It has been renamed to: %s.", "vet", "govet").
+		OnWarnf("The linter name %q is deprecated. It has been renamed to: %s.", "vetshadow", "govet").
+		OnWarnf("The linter name %q is deprecated. It has been renamed to: %s.", "logrlint", "loggercheck").
+		OnWarnf("The linter name %q is deprecated. It has been split into: %s.", "megacheck", "gosimple, staticcheck, unused").
+		OnWarnf("The linter name %q is deprecated. It has been renamed to: %s.", "gas", "gosec")
```

Comment:
Do these assertions work as expected? I would've thought the test would fail since the arguments to `mock.Call` are not the same as from the code anymore but it seems like the test passed? 🤔

Reply:
Yes but weirdly not in the CI...

If I run the test locally it fails as expected because I forget to update the test.

Reply:
It's a side effect of the `GL_TEST_RUN` inside the `test` target.

Reply:
Fixed.